src/JustEat.StatsD/IDisposableTimer.cs

```diff
@@ -1,9 +1,15 @@
-﻿using System;
+using System;
 
 namespace JustEat.StatsD
 {
+    /// <summary>
+    /// Defines a timer which is disposable.
+    /// </summary>
     public interface IDisposableTimer : IDisposable
     {
+        /// <summary>
+        /// Gets the name of the statsd bucket associated with the timer.
```

Comment:
"Gets or sets the name" ? It's writable so that you can alter the stat name based on eg. http response code inside the block being timed.

Reply:
Good spot.

Reply:
One day I'll convince people StyleCop's a good idea, and that would've found this for me.